### PR TITLE
For review by Klynton - Smoke testing

### PR DESCRIPTION
**Pull request by ErinBuckley**
For review by Klynton - initial commit for smoke test .md file

### CODE REVIEW

File: courses/puppet/smoke_test.md [hunk 1]
```diff
@@ -1,22 +1,54 @@
...
 
-Wkipedia
+After basic code validation, the next level of testing is smoke testing.
+The goal of smoke testing is to verify that your module runs and does what you want it to do.  
 
-Smoke testing (software): trying the major functions of software before carrying out formal testing
+At the end of this course you will be able to:
 
-In computer programming and software testing, smoke testing (also confidence testing, sanity testing[1]) is preliminary testing to reveal simple failures severe enough to reject a prospective software release. A subset of test cases that cover the most important functionality of a component or system is selected and run, to ascertain if crucial functions of a program correctly work.[2][1]:37 When used to determine if a computer program should be subjected to further, more fine-grained testing, a smoke test may be called an intake test.[1]:25
+* Set up dies?
```
Comment (klynton) [inline courses/puppet/smoke_test.md]: ?

Comment (ErinBuckley) [inline courses/puppet/smoke_test.md]: Where was this? In the course content? weird - probably me messing up with
freaking git again

Comment (klynton) [inline courses/puppet/smoke_test.md]: Yep, it's in the content.

**Comment by ErinBuckley** [comment]
Closing this myself - there is not content to review

**Pull request merged**